coops-ui/src/components/Dashboard/CoaWarningDialog.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
This dialog already exists in 'master'. Please don't clobber it as I made some changes to it in another PR.